### ISSUE TITLE
minifier: bad ouput case with 1.2.73

### ISSUE THREAD

**Issue by kapouer** [swc-project/swc#2010]
```
function testme(a, b) {
	const t = Object.assign(a, b);
	[a, b].forEach(v => global.func2(v, t));
	if (t.test) b.test = t.test;
}
```
gives
```
function testme(a,b){var c=Object.assign(a,b)[a,b].forEach(function(d){return global.func2(d,c)}),c.test&&(b.test=c.test)}
                                                                                                   ^
Uncaught SyntaxError: Unexpected token '.'
```
with config
```
{
	"jsc": {
		"loose": true,
		"minify": {
			"compress": true,
			"mangle": true
		}
	},
	"minify": true
}
```

**Comment by kapouer** [comment]
might very well be already fixed by #2006

**Comment by kapouer** [comment]
Yep, fixed
